### PR TITLE
Add test cases for kuiper integration-test

### PR DESCRIPTION
**Pull request by cherrycl**
Add test cases for Kuiper integration-test

#231
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Comment by cherrycl** [comment]
Test cases are updated.

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    Given Create A Stream
+    And Create A Rule With MQTT Action To Receive Data
+    And Create Device For device-virtual With Name kuiper-mqtt-device
+    When Get device data by device kuiper-mqtt-device and command ${command}
+    Then Device data has recevied by mqtt subscriber
+    [Teardown]  Run Keywords  Delete device by name kuiper-mqtt-device
+                ...      AND  Delete Rules
+                ...      AND  Delete Stream
+
+Kuiper002 - Add a new rule to update device command
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot:26]: `Add a new rule to execute set command`

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+*** Variables ***
+${SUITE}         Export By Kuiper Rules
+${LOG_FILE_PATH}          ${WORK_DIR}/TAF/testArtifacts/logs/export_by_kuiper_rules.log
+
+*** Test Cases ***
+Kuiper001 - Add a new rule and export to MQTT
+    Given Create A Stream
+    And Create A Rule With MQTT Action To Receive Data
+    And Create Device For device-virtual With Name kuiper-mqtt-device
+    When Get device data by device kuiper-mqtt-device and command ${command}
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot:20]: `When Execute get command ${command} to trigger kuiper-mqtt-device`

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    Then Device data has recevied by mqtt subscriber
+    [Teardown]  Run Keywords  Delete device by name kuiper-mqtt-device
+                ...      AND  Delete Rules
+                ...      AND  Delete Stream
+
+Kuiper002 - Add a new rule to update device command
+    Given Create A Stream
+    And Create Rules With REST Action To Set ${resource} Value ${resource_value}
+    And Create Device For device-virtual With Name kuiper-edgex-device
+    When Set device data by device kuiper-edgex-device and command ${command}
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot:30]:
```
When Execute get command ${command} to trigger kuiper-mqtt-device
And kuiper-mqtt-device execute set command ${command} according to the rule
```

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot (hunk 1 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot]: Updated.

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot (hunk 2 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot]: Updated.

File: TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot (hunk 3 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot]: Updated.

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**